lib/rabbitmq/cli/ctl/commands/cluster_status_command.ex

```diff
@@ -33,6 +33,8 @@ defmodule RabbitMQ.CLI.Ctl.Commands.ClusterStatusCommand do
         case :rabbit_misc.rpc_call(node_name, :rabbit_mnesia, :cluster_nodes, [:running]) do
           {:badrpc, _} = err ->
             err
+          {:error, {corrupt_or_missing_cluster_files, _, _}} = err ->
```

Comment:
Note: I don't know Elixir very well 😄

I can see that the error tuple atoms are not prefixed with `:` in the stack trace (#199):

```
{error,{corrupt_or_missing_cluster_files,{error,enoent},{error,enoent}}}}
```

In the code here, I see that the atom `:error` uses a colon. Does this mean that `corrupt_or_missing_cluster_files` should also use one?